common/config/rush/common-versions.json

```diff
@@ -1,58 +0,0 @@
-/**
- * This configuration file specifies NPM dependency version selections that affect all projects
- * in a Rush repo.  For full documentation, please see https://rushjs.io
- */
-{
-  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/common-versions.schema.json",
-  /**
-   * A table that specifies a "preferred version" for a dependency package. The "preferred version"
-   * is typically used to hold an indirect dependency back to a specific version, however generally
-   * it can be any SemVer range specifier (e.g. "~1.2.3"), and it will narrow any (compatible)
-   * SemVer range specifier.  See the Rush documentation for details about this feature.
-   */
-  "preferredVersions": {
-    /**
-     * When someone asks for "^1.0.0" make sure they get "1.2.3" when working in this repo,
-     * instead of the latest version.
-     */
-    // "some-package": "1.2.3"
-
-    // Pinning a dependency here forces it to show up in the generated common/temp/package.json,
-    // which reduces the chance of duplicates being added to the dependency tree and can help fix
-    // issues with missing peer dependencies.
-    "@types/react": "16.8.11",
-    "@types/react-dom": "16.8.4",
-    "@types/webpack-env": "1.13.9",
-    "css-loader": "^0.28.7",
-    // enzyme@3.10 causes issues with asAsync test in utilities
-    "enzyme": "~3.9.0",
-    "jest": "24.1.0",
-    // React deps are pinned to prevent Invalid Hook Warnings with duplicate React deps.
-    // https://reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react
```

Comment:
I'm particularly worried about React here. I needed to add this because React Hooks (per the comment) will [completely blow up](https://reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react) if there are multiple copies or versions of React present. This can be verified by checking all of the deployed sites to make sure everything still works.

Reply:
I hadn't seen it blow up in our demo yet... in fact, that's what yarn does - it hoists these deps on the consumer end.